lib/theming/src/create.ts

```diff
@@ -110,76 +115,85 @@ const darkSyntaxColors = {
   blue2: '#00009f',
 };
 
-export const create = (vars: ThemeVar, rest?: Rest): Theme => ({
-  base: vars.base,
-  color: createColors(vars),
-  background: {
-    app: vars.appBg || background.app,
-    content: vars.appContentBg || color.lightest,
-    hoverable: vars.base === 'light' ? 'rgba(0,0,0,.05)' : 'rgba(250,250,252,.1)' || background.hoverable,
-
-    positive: background.positive,
-    negative: background.negative,
-    warning: background.warning,
-  },
-  typography: {
-    fonts: {
-      base: vars.fontBase || typography.fonts.base,
-      mono: vars.fontCode || typography.fonts.mono,
+function pick<B, K extends keyof B>(b: B, key: K): B[K] {
+  return b[key];
+}
+
+export const create = (vars: ThemeVar, rest?: Rest): Theme => {
+  const inherit: ThemeVar = { ...vars, ...(base[vars.base] || base.light), ...base.light };
+  const p = pick.bind(null, inherit);
+
+  return {
+    base: vars.base,
+    color: createColors(inherit),
+    background: {
+      app: p('appBg'),
```

Comment:
I don't really understand why this isn't just `inherit.appBg`?

Reply:
hmm, yeah